src/components/modal/modal.html

```diff
@@ -12,19 +12,17 @@
         </svg>
       </button>
     </div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id accumsan augue. Phasellus consequat augue vitae tellus
+      tincidunt posuere. Curabitur justo urna, consectetur vel elit iaculis, ultrices condimentum risus. Nulla facilisi.
+      Etiam venenatis molestie tellus. Quisque consectetur non risus eu rutrum. </p>
+  </div>
 
-    <div class="bx--modal-content">
```

Comment:
Are we eliminating this `<div>`?

Reply:
Looks like the modal html file was edited in error? If I revert looks good in IE.

Reply:
Agree, looks like this was edited in error?

Reply:
It certainly was 🙂